src/schemas/book.py

```diff
@@ -6,21 +6,37 @@
 
 class BookSchema(BaseModel):
     id: Optional[int] = None
-    title: str = Field()
+    title: str = Field(min_length=1)
     date_of_publication: str = Field()
-    isbn: str = Field()
+    isbn: str = Field(min_length=10, max_length=13)
     description: str = Field(min_length=0, max_length=200)
     language_id: int = Field()
-    author_ids: List[int] = Field()
-    genre_ids: List[int] = Field()
+    author_ids: List[int] = Field(min_items=1)
+    genre_ids: List[int] = Field(min_items=1)
 
     @validator("date_of_publication")
     def validate_my_date(cls, value):
+        if len(value) < 0:
+            raise ValueError("Can`t be an empty Field")
         input_date = date.fromisoformat(value)
         if input_date > date.today():
             raise ValueError("Date can't be greater than today")
         return value
 
+    @validator("title")
```

Comment:
This explicit validation is not required if you have already specified min_length = 1 above